config/locales/en.yml

```diff
@@ -1242,10 +1242,11 @@ en:
           <span>Make sure to <a href="https://docs.openshift.com/container-platform/latest/dev_guide/routes.html#creating-routes">add the correct route</a></span>
         endpoint_apicast_2_html: >
           Public address of your API gateway in the %{environment_name} environment.
-        api_backend_https: "Private address of your API that will be called by the API gateway."
+        api_backend_https: |
```

Comment:
can't we just use this message in backend edit?

Reply:
Yes, we can